cli/ballerina-cli/src/main/resources/cli-help/ballerina-openapi.help

```diff
@@ -86,6 +87,9 @@ OPTIONS
             This option can be used in the service generation to generate a low-level service
             without any data-binding logic.
 
+        --with-status-code-binding
+            This option can be used in the client generation to generate the client methods with status code response binding.
```

Comment:
May be too long. I can't remember the length exactly, but since this is CLI help text, we stick to a specific length. Shall we update at least to be in-line with the rest of this help text?

Reply:
Fixed via [91616f3](https://github.com/ballerina-platform/ballerina-lang/pull/42559/commits/91616f3ec6c477d6ebdc8d09e1865918db12ad18)